docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -2,17 +2,20 @@
 
 = Comparing configuration files between deployments
```

Comment:
Comment from Pedro: The title "Comparing configuration files between deployments" may lead users to anticipate instructions on utilizing a specific tool for this purpose. I think we should at least introduce a basic example.